src/test/ts/part3/Part3Ex1Test.ts

```diff
@@ -0,0 +1,99 @@
+import { after, before, describe, it } from '@ephox/bedrock-client';
+import { Attribute, Insert, Remove, SelectorFind, SugarBody, SugarElement } from '@ephox/sugar';
+
+import { assert } from 'chai';
+import tinymce from 'tinymce';
+
+/*
+In this part we're going to discuss working with the editor in a little more
+detail. We'll look at this under the pretense of writing tests for editor
+functionality, and try to sneak in some knowledge about the editor itself as we
+go.
+ */
+
+describe('Part3Ex1Test', () => {
+  it('looks like an editor', () => {
+    /*
+    For now, let's stick with a basic test. Does an editor exist? We can figure
+    this out using the TinyMCE global - by importing it directly from
+    node_modules, we can avoid the script tag and the API key - in the docs for
+    our customers we refer to this as "self-hosting"
+     */
+    assert.isNotEmpty(tinymce.editors);
```

Comment:
```suggestion
    assert.isNotEmpty(tinymce.get());
```
`editors` is a deprecated API so we probably should avoid mentioning it :shrug: